phpstan.neon

```diff
@@ -0,0 +1,36 @@
+includes:
+	- phpstan-baseline.neon
+
+parameters:
+    inferPrivatePropertyTypeFromConstructor: true
+    checkMissingIterableValueType: false
+    checkUninitializedProperties: true
+    checkGenericClassInNonGenericObjectType: false
```

Comment:
this one is a pain to fix, and because we may change how the proxy stuff is done, the pain is not worth it